.github/workflows/smoke.yml

```diff
@@ -287,3 +287,29 @@ jobs:
       run: |
         export PERFORMANCE_TEST_MODE=${{ matrix.mode }}
         sudo -E make smoke-performance
+
+  failover-test:
+    runs-on: ubuntu-latest
+    needs: [contrib-build, nydus-build]
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Download Nydus
+      uses: actions/download-artifact@master
```

Comment:
Please use `v3`, `master` is unstable, and `v4` has some bugs that need to be fixed.

Reply:
Fixed.